pkgs/by-name/sy/systemd-manager-tui/package.nix

```
@@ -0,0 +1,30 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
Fixed-point lambdas are now supported for Rust builders and are in general preferred over rec whenever possible